packages/types/lib/plans/http.api.ts

```diff
@@ -23,3 +23,12 @@ export interface PlanDefinition {
     hidden?: boolean;
     flags: Omit<Partial<DBPlan>, 'id' | 'account_id'>;
 }
+
+export type GetPlans = Endpoint<{
+    Method: 'GET';
+    Path: '/api/v1/plans';
+    Querystring: { env: string };
+    Success: {
+        data: PlanDefinition[];
```

Comment:
do we need the data field. Can we return the list directly?

Reply:
It's mostly for consistency. My experience tells me that if we allow ourselves to do that, then at some point we will say "there is only one object when we get one resource, so let's return it directly," and this object might have an `error` prop, which makes error detection error-prone